docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -159,6 +159,26 @@ See the https://playwright.dev/docs/api/class-page#page-get-attribute[`page.getA
 <2> Use the assertion library provided by the Synthetics agent to look for the
 expected value. See https://jestjs.io/docs/expect[Jest expect docs] for more information.
 
+[discrete]
+[[synthetics-create-test-script-recorder]]
+=== Synthetics Recorder
+
+beta[]
+
+// In what circumstances should you use Synthetics Recorder?
+Use the Synthetics Recorder to generate code for one or more steps by interacting with a web page directly.
+
+// Some high-level details on what it does
+The recorder launches a special https://www.chromium.org/Home/[Chromium browser] that will listen to each interaction you have with the web page and record them internally.
```

Comment:
Not sure about the `special` wording here, AFAIK it's a normal Chromium browser that is used by Playwright.
```suggestion
The recorder launches a https://www.chromium.org/Home/[Chromium browser] that will listen to each interaction you have with the web page and record them internally using Playwright.
```